packages/angular/src/tracing.ts

```diff
@@ -49,6 +56,23 @@ export function routingInstrumentation(
 
 export const instrumentAngularRouting = routingInstrumentation;
 
+/**
+ * A custom BrowserTracing integration for Angular.
+ */
+export function browserTracingIntegration(
```

Comment:
l: Just thought we should add a line here as it's quite crucial to set up performance monitoring correctly in Angular

```suggestion
/**
 * A custom BrowserTracing integration for Angular.
 *
 * Use this integration in combination with `TraceService`
 */
export function browserTracingIntegration(
```